sys-devel/qconf/qconf-2.4.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit qmake-utils
+
+DESCRIPTION="./configure like generator for qmake-based projects"
+HOMEPAGE="https://github.com/psi-plus/qconf"
+SRC_URI="http://psi-im.org/files/qconf/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd"
+IUSE=""
+
+# There is no one to one match to autotools-based configure
+QA_CONFIGURE_OPTIONS=".*"
```

Comment:
is this really still needed?

Reply:
I don't mind to remove this if it's fine to have warnings in the log.

Reply:
I overlooked the `econf` call, what exactly is the message?

Reply:
* QA Notice: Unrecognized configure options:
 * 
 * 	--build=x86_64-pc-linux-gnu
 * 	--host=x86_64-pc-linux-gnu
 * 	--mandir=/usr/share/man
 * 	--infodir=/usr/share/info
 * 	--sysconfdir=/etc
 * 	--localstatedir=/var/lib
 * 	--libdir=/usr/lib64

Reply:
https://github.com/psi-plus/qconf/blob/master/configure that thing is not an autoconf-generated `configure` script, instead its a crappy homebrew NIHism. Here's my suggestion: nix `QA_CONFIGURE_OPTIONS`, and instead of calling `econf`, call `./configure` and add a `|| die` at the end (maybe also throw in `--prefix` and what not).

Reply:
yes I get that, but trying to emulate autoconf and only doing a halfarsed job (no `--libdir`, seriously?) means `econf` is generally out of the picture. I know the communication coming from Gentoo isn't always stellar and often full of internal tensions. Could you give me a link/paste/something to show where someone told you to try and use `econf` at all costs?

Reply:
libdir is supported for projects which declare they need it in their *.qc file.
take a look on net-im/psi for example.
Regarding econf, I remember this in some net-im/psi thread in bug tracker. If I find  it, I'll let you know.

As one of developers of qconf, I can add ignore to autoconf options or at least print warnings for them to  stdout instead of stderr.

Reply:
I didn't find this in the bug tracker. But I definitely remember some wishes with this regard.
I'll convert the ebuilds to direct configure calls shortly. although I don't like the idea.

Reply:
Add a comment that it isn't an autotools-produced `configure` script, `econf` is really only for autoconf